docker/kubernetes/ray/images/run.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+function run_head() {
+    ray start --head --redis-port=6379
```

Comment:
This command starts multiple Redis servers. The ports of the other Redis servers can be specified with `--redis-shard-ports` (in a comma separated list).

This command also starts an object manager whose port can be specified with `--object-manager-port`.

Reply:
ok, thanks.

Reply:
I will read your code and docs more carefully.

Reply:
sorry, last week i was busy on kubeless.Now let's continue.
--redis-shard-ports specify the other redis servers but how can ray head use the redis server?

Reply:
I plan to use HostNet mode for both ray head and redis. But this means redis and ray head must be in the same host.

Reply:
Can I change some script in python/ray? I want one container have just one process.